interview-questions.md

```diff
@@ -14,44 +14,44 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git stands for global information tracker. It is a tool used by programmers to effectively and efficiently work on code. The structure consists of a main branch that contains all the merged code and side branches that programmers can work on without affecting the main branch. Multiple groups of programmers can work on different branches for different areas of code so that not one group is left waiting for another to complete their code. While git is the tool that allows programmers to code together, Github is the platform/workspace. Code can be used by git to send to Github so that other programmers can see the work that another has done. On Github, we can review the codes pushed onto branches, create pull requests, and merge the side branches to the main branch.
 
-Researched answer:
+Researched answer: Git is a version control system that allows us to keep track of our code history. In Github, we can make and manage repositories. While we need git to use Github, we do not need Github to use git. There are platforms other than Github that we can use git with. Github is simply the first, which is why when we send code to Github using terminals, it is referred to as origin.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: A Boolean value is true or false, and the operators == (loosely equal), === (strictly equal), > (greater than), >= (greater than or qual to), < (less than), <= (less than or qual to), !== (strictly not equal) will return a Boolean value.
 
-Researched answer:
+Researched answer: There is also the != (loosely not equal) operator. We can combine conditions using these operators with && (AND) and || (OR). With &&, both conditions must be true for it to be true. With ||, one condition must be true for it to be true.
 
```

Comment:
This was looking at the categories of operators that produce a boolean value. You have examples of all three: equality, relational, and logical